test/test.py

```diff
@@ -179,7 +194,32 @@ def test_preservation_storage_bundle(self):
                           diseases={'normal'},
                           project_roles={'Human Cell Atlas wrangler', None, 'external curator'},
                           storage_methods={'frozen, liquid nitrogen'},
-                          preservation_methods={'cryopreservation, other'})
+                          preservation_methods={'cryopreservation, other'},
+                          library_construction_methods={'Smart-seq2'})
+
+    def test_dissociation_protocol_fields(self):
```

Comment:
Do we need this test still, now that yu removed the changes to DissociationProtocol?

Reply:
Correct, the tests for DissociationProtocol and CollectionProtocol fields are no longer needed. I've just committed change 4d15ecc to remove them.